api/src/unraid-api/graph/resolvers/display/display.service.ts

```
@@ -0,0 +1,140 @@
+        return {
+            ...states.default,
+            icon: serverCase,
+        };
+    }
+
+    private async getDisplayConfig() {
+        const filePaths = getters.paths()['dynamix-config'];
+
+        const state = filePaths.reduce<Partial<DynamixConfig>>((acc, filePath) => {
+            const state = loadState<DynamixConfig>(filePath);
+            return state ? { ...acc, ...state } : acc;
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)